meetups/brno.yml

```
@@ -365,3 +365,12 @@
   date: 2022-04-02
   time: 9:00 – 17:00
   place: *fi-muni
+  
+ - name: PyWorking: Automatické testy
```

```suggestion
- name: Testovací PyWorking
```
There is an additional space at the beginning that shouldn't be there + I've got a parsing error due to the colon in the PyWorking's name, hence proposing a name change.